volumes/ecocounter/tables/sites.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE ecocounter.sites (
```

Comment:
Can we add comments for each table description what it stores? Is site_id the unique location id? Still have a slight confusion of what replaced_by contains

Reply:
is it an old location replaced by a new location ?

Reply:
I've changed the name of the column to try to be more clear. The `replaced_by_site_id` is the `site_id` of the new replacement site, if there is one. Some new / replacement sensors got installed in a few sites and instead of having the same `site_id`, they have a new one even though they're in the same physical location.

Reply:
I added a comment trying to explain what a site is. If it still seems a bit vague, that may be because it is a bit.